src/components/MetadataDisplay/MetadataDisplay.js

```diff
@@ -1,20 +1,17 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import bem from '@fesk/bem-js';
-import formatDate from 'date-fns/format';
+// import formatDate from 'date-fns/format';
 import {
   Card,
   CardContent,
-  CardHeader,
   Typography,
-  CardActions,
   IconButton,
   Grid,
 } from '@material-ui/core';
 import { Edit } from '@material-ui/icons';
 
-const displayTime = time =>
-  formatDate(time, time >= 3600000 ? 'hh:mm:ss' : 'mm:ss');
+// const displayTime = time =>
+//   formatDate(time, time >= 3600000 ? 'hh:mm:ss' : 'mm:ss');
```

Comment:
Best to delete the lines

Reply:
https://github.com/digirati-co-uk/timeliner/pull/105/commits/e9edc1502dad5b042633e8eab03de6d80a65a0c5

Reply:
https://github.com/digirati-co-uk/timeliner/pull/105/commits/525fd85f709f47746be00269b2dc6ebf94e50a33